scripts/install_submitter.py

```
@@ -0,0 +1,619 @@
+
+        <install_dir>/
+        ├── plug-ins/
+        │   └── DeadlineCloudForVRED.py
+        ├── scripts/
+        │   └── deadline/
+        │       └── vred_submitter/
+        │           ├── __init__.py
+        │           ├── _version.py
+        │           ├── vred_submitter.py
+        │           └── ... (other submitter modules)
+        └── python/
+            └── modules/
+                └── ... (dependency packages)
+        """
+        files: dict[str, List[Path]] = {"plugin": [], "scripts": [], "dependency_bundle": []}
+
+        # Plugin files
+        plugin_dir = self.package_root / "vred_submitter_plugin" / "plug-ins"
+        if plugin_dir.exists():
+            files["plugin"] = list(plugin_dir.rglob("*.py"))
+
+        # Scripts (src/deadline/vred_submitter directory)
+        scripts_dir = self.package_root / "src" / "deadline" / "vred_submitter"
+        if scripts_dir.exists():
+            files["scripts"] = list(scripts_dir.rglob("*"))
+
+        # Dependency bundle - build if not exists
+        dependency_bundle_dir = self.package_root / "dependency_bundle"
+        if not dependency_bundle_dir.exists():
+            logger.info("🧱 Building dependency bundle...")
+            if build_deps_bundle is not None:
+                try:
+                    # Change to package root directory for deps_bundle to work correctly
+                    original_cwd = os.getcwd()
+                    os.chdir(self.package_root)
+                    build_deps_bundle()
+                    os.chdir(original_cwd)
+                    logger.info("🧱 Dependency bundle built successfully")
+                except Exception as e:
+                    logger.warning("Failed to build dependency bundle: %s", e)
+                    os.chdir(original_cwd)
+            else:
+                logger.warning("Dependency bundle not found and build_deps_bundle not available.")
```

The deps_bundle.py script should be right beside this script, so shouldn't we just exit early during initialization if we can't find it? I'd rather us just exit early rather than quietly fail and continue along the installation

---

Right! I have updated it to fail fast if it's not found.